cdap-ui/app/cdap/services/StatusFactory.js

```diff
@@ -108,7 +108,7 @@ const getRequestInfo = () => {
   };
   let requestInfo = {
     credentials: 'include',
-    mode: 'no-cors',
+    // mode: 'no-cors',
```

Comment:
Same here. Was this change intended to be in this pull request?

Reply:
No. I reverted this change.